packages/playkit-js-scripts/bin/kcontrib.js

```diff
@@ -25,13 +25,9 @@ let script = scriptIndex === -1 ? args[0] : args[scriptIndex];
 const nodeArgs = scriptIndex > 0 ? args.slice(0, scriptIndex) : [];
```

Comment:
General comment:
The generated folder still contain the webpack files: 
1. webpack.common.js
2. webpack.dev.js
3. webpack.prod.js

Reply:
I will remove them when serve command will be transfered to cli
FEV-439